OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/toolbar/SpellCheck.java

```diff
@@ -0,0 +1,49 @@
+/* 
+ * Copyright (c) 2017 White Hsu
+ * 
+ * See the file LICENSE for copying permission.
+ */
+package latexstudio.editor.toolbar;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.text.BadLocationException;
+import latexstudio.editor.EditorTopComponent;
+import latexstudio.editor.TopComponentFactory;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.awt.ActionRegistration;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle.Messages;
+
+@ActionID(
+        category = "Edit",
+        id = "latexstudio.editor.toolbar.SpellCheck"
+)
+@ActionRegistration(
+        iconBase = "openlatexstudio/icons/spellcheck.png",
+        displayName = "#CTL_SpellCheck"
+)
+@ActionReferences({
+    @ActionReference(path = "Menu/Edit", position = 250),
+    @ActionReference(path = "Toolbars/Comment", position = 3433)
+})
+@Messages("CTL_SpellCheck=Spell Check")
+public final class SpellCheck implements ActionListener {
+
+    private final EditorTopComponent etc = new TopComponentFactory<EditorTopComponent>()
+            .getTopComponent(EditorTopComponent.class.getSimpleName());
+    
+    @Override
+    public void actionPerformed(ActionEvent e) {    
+        
+        try {
+            etc.spellCheckAllText();
+        } catch (BadLocationException ex) {
+            Exceptions.printStackTrace(ex);
+        }      
+        
+        etc.getEditorState().setDirty(true);        
```

Comment:
@sebbrudzinski
Thanks for feedback. I've updated codes accordingly. If it's OK, please review and accept my change. Then I'll create new items and work on more enhancements for this Spell Check feature. Thank you. :-)